### PR TITLE
Fixing the `bench-bot` after substrate changes

### PR DESCRIPTION
**Pull request by georgesdib**
Needed after substrate `benchmark` sub-command change
PR paritytech/substrate#11164

I believe this is what is needed now, but I am not sure.

### CODE REVIEW

File: bench.js [hunk 1]
```diff
@@ -227,6 +229,7 @@ var PolkadotRuntimeBenchmarkConfigs = {
       "--features=runtime-benchmarks",
       "--",
       "benchmark",
+      "pallet",
```
Comment (ggwpez) [inline bench.js:232]: It could also be needed in Lines 328 and 419. Although I dont know what they do :laughing:   
@joao-paulo-parity maybe you know.

**Review by ggwpez — APPROVED** [review]
Thanks we were just talking about this!

File: bench.js (hunk 1 above)
Comment (joao-paulo-parity) [reply bench.js:232]: From what I understand those are custom "run whatever" commands which could have a different subcommand other than `pallet`, so I think they can stay as they are. I don't know who's using them anyhow, perhaps @shawntabrizi can shed some light here.

In any case that should not block this PR and we can add it later if necessary.

**Comment by joao-paulo-parity** [comment]
Thanks for the PR @georgesdib, let's merge this

**Pull request merged**